pkpdapp/pkpdapp/urls.py

```diff
@@ -27,4 +27,12 @@
     path('generic/', views.GenericView.as_view(), name='generic'),
```

Comment:
Could you sort this alphabetically? And maybe separate out the dash and generic url. The dash one has to be there to use dash apps in django but we never really access this url directly, and the generic one is only temporary. 

Maybe that makes it easier to understand when we keep extending urls.

Reply:
sure, this is done.